parser/v2/types.go

```diff
@@ -173,7 +173,7 @@ type Whitespace struct {
 func (ws Whitespace) IsNode() bool { return true }
 
 func (ws Whitespace) Write(w io.Writer, indent int) error {
-	if ws.Value == "" || !strings.Contains(ws.Value, "\n") {
+	if ws.Value == "" || strings.Contains(ws.Value, "\n") {
```

Comment:
How yes and also I edit those lines cause I think there was an error and this error make my test don't work.
Basically `writeNodes` would sometime write a space cause in the original text we had space. 
```html
<strong>Souffle</strong> chocolate
```
The issue was that condition block the write of the tiny whitespace cause that tiny whitespace don't have a line line.
But since it's always `writeNodes` that is responsible for new line.
I think this is an error cause we should never write new line everywhere except for `writeNodes`.